raft/raft.go

```diff
@@ -582,11 +582,6 @@ func (r *raft) Step(m pb.Message) error {
 		}
 		return nil
 	}
-	if m.Type == pb.MsgTransferLeader {
```

Comment:
It's there because previously we ignore `MsgTransferLeader` as local message in follower, candidate. But now we don't ignore and forward it to leader.